packages/core/src/external/aws/s3.ts

```diff
@@ -27,15 +33,21 @@ export async function getSignedUrl({
 
 export class S3Utils {
   public readonly _s3: AWS.S3;
```

Comment:
I think this should be marked as deprecated, right? If I understand it correctly, we now have v2 and v3 of AWS SDK on the same file, right?